java/data-management/strings-iii/altering-format-string-output-by-changing-a-format-specifier-s-argument-index.md

```diff
@@ -41,29 +37,38 @@ String s = String.format(
   hippo);
 ```
 
-Here, `"%s %s"` is a format string, and `%s` is a format specifier. The value of `s` is `"hungry hippo"`.
+Here, `"%s %s"` is a format string, and `%s` is a format specifier. The `%s` specifier will be replaced with the first string given as an argument.
 
-Modify the order that the arguments appear in the format string by specifying an argument index in the format specifiers.
+In this example, `s` will be formatted to `"hungry hippo"`.
```

Comment:
Let's name the `String s` as `String result` or something more differentiable? There's a lot of "s" sounds already :D